.github/workflows/nightly-linux.yml

```diff
@@ -0,0 +1,34 @@
+name: Nightly Linux
+
+on:
+  # Automatically run at the end of every day.
+  schedule:
+    - cron: '0 0 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || format('{0}-{1}', github.ref, github.run_number) }}
```

Comment:
I don't think cron runs ever have `github.head_ref` set; we should always use `github.ref`. Same for `nightly-android.yml` which I assume this was taken from.

Reply:
ah, yep, it was copied from there. Which was copied from ci.yml originally. I've moved to just github.ref